hpx/include/serialization.hpp

```diff
@@ -9,7 +9,6 @@
 #include <hpx/runtime/serialization/serialization_fwd.hpp>
 #include <hpx/runtime/serialization/serialize.hpp>
 
-#include <hpx/runtime/serialization/allocator.hpp>
```

Comment:
@sithhell it wasn't used (allocator was serialized as empty type). Moreover the code was incorrect as it didn't have split macro